whitelabelco/sofo-foods
# Cp/feat/new about hero image

### Description 💻

Resolves: #220 & #221

### Implementation 💡

- replace image of corporate office on the About page with infographic
- set custom font-weight and change letter spacing 


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- click on About Us
- confirm infographic 

### Screenshots/Demos 📷

### Checklist ✅
